src/pages/LandingPage.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+import LayoutContainer from "../components/LayoutContainer";
+import { Paths } from "../constants";
+import { UnauthenticatedPageProps } from "../types";
+
+const LandingPage = ({
+  setSessionUser
+}: UnauthenticatedPageProps): JSX.Element => {
+  return (
+    <LayoutContainer
+      activePath={null}
+      sessionUser={null}
+      setSessionUser={setSessionUser}
+    >
+      <h1>Try out Reel Politik</h1>
```

Comment:
😄 let's add some copy or something here at some point to help our FTUX!

Reply:
Gonna leave that for another PR if that's alright